### ISSUE TITLE
Light theme?

### ISSUE THREAD

**Issue by mykeehu** [Vali-98/ChatterUI#159]
I like this app, but I can't use it for a long time because of the dark theme. I couldn't find a light theme when changing the colors, but my eyes have a hard time with dark backgrounds with white letters.
Will there be a light theme for the app?

**Comment by Slug-Cat** [comment]
Have you tried inverted color mode?

**Comment by mykeehu** [comment]
Nice trick, but it is very inconvenient to have to change the system every time I start the application.

**Comment by Vali-98** [comment]
One goal is to do a proper re-theming of the app, which should also introduce some built-in themes alongside seperation of dark and light themes.

**Comment by Puron83** [comment]
Yeah, I would love a light theme. Reading on the dark dark background really stresses my eyes

**Comment by Vali-98** [comment]
Closed in favor of #214
